simulation.cpp

```diff
@@ -51,13 +51,13 @@ simulation::simulation(all_params params):
 
 const std::vector<double>& simulation::get_inds_input() const
```

Comment:
The more I look at it the more I wish this function was free and wild roaming in the endless planes outside its class scope.
Think about it, you might make a class and a function happy and not have to use `*this`

Reply:
🏇